package.json

```diff
@@ -73,7 +74,10 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "lint-staged"
     }
+  },
+  "lint-staged": {
+    "*.ts": "prettier --write && tslint -p tsconfig.json"
```

Comment:
Yeah, you're right. Since, we've been using `pretty-quick`, let's just replace the line to `"pretty-quick --staged && tslint -p tsconfig.json"`

Reply:
I've set it to just `pretty--quick` as `lint-staged` already grabs staged files. If this doesn't have the functionality we expect it's an easy update